sidebars.ts

```diff
@@ -76,8 +76,7 @@ const sidebars: SidebarsConfig = {
             "client/javascript-mono/ReactUsage",
             "client/androidClientSDK",
             "client/iosClientSDK",
-            "client/javascript-mono/NextJsAppRouterUsage",
-            "client/javascript-mono/NextJsPageRouterUsage",
```

Comment:
need to redirect both of these in docusaurus config to the new page.  no broken links